### PR TITLE
Toggle SELinux volume label based on OS version

### PR DESCRIPTION
**Pull request by charlieegan3**
**What this PR does / why we need it**:
We've been having an issue using the old label on the latest versions.

Source: https://docs.openshift.com/container-platform/3.6/install_config/persistent_storage/pod_security_context.html

> The volume will be given a type which is accessible by unprivileged containers. This type is usually container_file_t, which treats volumes as container content. Previously, the label specified was svirt_sandbox_file_t. This label is no longer used due to security concerns.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Use the container_file_t volume label on newer os versions.
```

### CODE REVIEW

**Comment by charlieegan3** [comment]
WIP while I test that `operatingsystemrelease` does what I expect

**Comment by charlieegan3** [comment]
I can run the quick verify command locally and it passes.

```
make -C puppet/modules/tarmak verify
```

For some reason the variable for the os release looks to not be getting set in the test infra run.

File: puppet/modules/kubernetes/manifests/kubelet.pp [hunk 1]
```diff
@@ -107,8 +107,13 @@
     $_ca_file = $ca_file
   }
 
+  $os_version = 0 + $facts["operatingsystemrelease"]
```
Comment (simonswine) [inline puppet/modules/kubernetes/manifests/kubelet.pp]: Can this part behave like the following:

- If fact is existing and os_version < 7.5 use `svirt_sandbox_file_t`
- Else use `container_file_t`

Also for comparing the version, can we please use that: https://github.com/jetstack/tarmak/blob/fee1c17ecb78942474047e0cb7815fc1105269f8/puppet/modules/prometheus/manifests/kube_state_metrics.pp#L19

**Comment by simonswine** [comment]
@charlieegan3 I think to reproduce it locally you might need to set this:

```
export FIXTURES_YML=.fixtures.yml.local
```

File: puppet/modules/kubernetes/manifests/kubelet.pp (hunk 1 above)
Comment (charlieegan3) [reply puppet/modules/kubernetes/manifests/kubelet.pp]: That's much nicer. I'd got it working with split & more mess but oh well - wish I'd known about this earlier :man_facepalming:

**Comment by charlieegan3** [comment]
I'm able to see the same failures if I run it with the fixtures variable (thanks Christian)

```
FIXTURES_YML=.fixtures.yml.local make -C puppet/modules/tarmak verify
```

Investigating now...

**Comment by simonswine** [comment]
Closed in favour of #211

**Pull request closed without merging**